src/features/testing/workflows/passive.md

```diff
@@ -0,0 +1,77 @@
+# Passive Type Workflows
+
+_For general documentation on utilizing the Workflows tab - click [here](../workflows.md)._
+
+---
+
+`Passive Workflows` take **requests** or **responses** as input. Their execution occurs in the "background" as you conduct your testing, extending the efficiency of your process.
+
+## Using Passive Workflows
+
+---
+
+Passive Workflows are **automatically triggered** when their specifications/conditions are met. These specifications/conditions are set by [Nodes](../../../concepts/nodes.md) and include prerequisites such as:
+
+- If the request/response is within a set [Scope](../../overview/scope.md).
+- If the request/response is a match according to [HTTPQL](../../../concepts/httpql.md) syntax.
+- If the prior Node's specification/condition evaluated to True or False (_Boolean value_).
+
+## Creating a New Passive Workflow: Applying Color to In-Scope GET Requests Workflow Example
```

Comment:
We should include the JSON has a `<details>` (hidden with an open arrow, we have a few examples in the repo) so people can follow along with the values.